src/sentry/tasks/post_process.py

```diff
@@ -181,15 +181,22 @@ def post_process_group(
     from sentry.utils import snuba
     from sentry.reprocessing2 import is_reprocessed_event
 
+    super_important_non_generic_ass_event_processing_store_cache_key = cache_key
```

Comment:
```suggestion
    processing_cache_key = cache_key
```
Less likely to get overwritten 🤷

Reply:
I'll go with it 👯